src/platform/forms-system/src/js/helpers.js

```diff
@@ -642,3 +644,50 @@ export function transformForSubmit(
 
   return JSON.stringify(withoutViewFields, replacer) || '{}';
 }
+
+/**
+ * Numerate the field names (keys) in a uiSchema object.
```

Comment:
Probably worth calling out in the description here that it intentionally avoids adding numbers to keys which begin with `ui:`.